lib/src/board_settings.dart

```diff
@@ -17,6 +17,18 @@ enum PieceShiftMethod {
   either;
 }
 
+/// Describes if and which pieces are displayed upside down (rotate by 180 degrees).
+///
+/// This is useful for implementing offline ("over the board") chess games.
+enum UpsideDownPieces {
+  // No pieces are upside down.
+  none,
+  // Opponent's pieces are displayed upside down.
+  opponent,
+  // Both sides' pieces are upside down.
+  all,
```

Comment:
Not such a big deal, but this new settings doesn't respect anymore the separation between what can change during a game vs what doesn't change (what settings are).

Because here you'd change the upside down value based on which side is to play for the `all` value.

So I'd suggest a slightly different approach:
```dart
enum PieceOrientationBehavior {
  /// Pieces are always facing user
  default,
  
  /// Opponent's pieces are upside down, for over the board play face to face.
  opponentUpsideDown,

  /// Piece orientation matches side to play, for over the board play where each user grab the device in turn
  sideToPlay,
}
```

This way you don't have to modify the settings, and the orientation logic is done on chessground side.